workspaces/services/guacamole-azure-dsvm/readme.md

```diff
@@ -0,0 +1,27 @@
+# Guacamole Target Resource Service bundle (Windows DSVM)
+
+This is a Target Resource Service template. It contains a Windows DSVM to be used by TRE researchers and to be connected using a [Guacamole server](https://guacamole.apache.org/).
```

Comment:
Not sure "target resource" is the right term we should be using. It doesn't make sense to be. "Service Child Resource Template" maybe?!

Reply:
Child-resource service template?

Reply:
Not sure the VM is a service. "User Resource Template" or "Service Resource Template" maybe.

Reply:
User Resource Template

Reply:
Lets stick with `User Resource Template` and `User Resource`. Agree?!